libraries/lib-strings/UnusedStrings.h

```diff
@@ -96,25 +96,6 @@ Some example strings are also given first, to document the syntax.
 // //////////////////////////////////////////////// End examples
 #endif
 
-
-// Crash & error report window
-
-XO("Problem Report for Audacity"),
-XO("Click \"Send\" to submit the report to Audacity. This information is collected anonymously."),
-XO("Problem details"),
-XO("Comments"),
-XC("&Send",       "crash reporter button"),
-XC("&Don't send", "crash reporter button"),
-// i18n-hint C++ programming exception
-XO("Exception code 0x%x"),
```

Comment:
This string had an i18n-hint comment that was not preserved in the cut-and-paste.  That should be done.  The comment must be on the line before the string.  This will guarantee that we preserve this comment in the audacity.pot file in future versions.  The comment helps the translator when the English string alone isn't completely self-explanatory.